tests/dsl/test_partial.py

```diff
@@ -95,7 +95,6 @@ class Summary(BaseModel):
     updates = 0
     for extraction in extraction_stream:
         if previous_summary is not None and extraction:
-            assert extraction.summary.startswith(previous_summary)
```

Comment:
I removed this since we only update the field once with the new `on` parsing strategy in jiter to support literals

Reply:
Relevant PR